File: csharp/AppEncryption/AppEncryption/Exceptions/KmsException.cs
using GoDaddy.Asherah.Crypto.Exceptions;

namespace GoDaddy.Asherah.AppEncryption.Exceptions
{
    public class KmsException : AppEncryptionException
    {
        /// <summary>
        /// Initializes a new instance of the <see cref="KmsException"/> class. This signals that a
        /// <see cref="GoDaddy.Asherah.AppEncryption.Kms.KeyManagementService"/> exception has occured.
        /// </summary>
        ///
        /// <param name="message">The detailed exception message.</param>
        public KmsException(string message)
            : base(message)
        {
        }
    }
}

File: java/app-encryption/src/main/java/com/godaddy/asherah/appencryption/exceptions/KmsException.java
package com.godaddy.asherah.appencryption.exceptions;

public class KmsException extends AppEncryptionException {
  /**
   * Creates a new {@code KmsException}. This signals that a
   * {@link com.godaddy.asherah.appencryption.kms.KeyManagementService} exception has occurred.
   *
   * @param message The detailed exception message.
   */
  public KmsException(final String message) {
    super(message);
  }
}
